deepreg/model/loss/label.py

```diff
@@ -164,13 +162,11 @@ def dice_score(y_true: tf.Tensor, y_pred: tf.Tensor, binary: bool = False) -> tf
     if binary:
         y_true = tf.cast(y_true >= 0.5, dtype=tf.float32)
         y_pred = tf.cast(y_pred >= 0.5, dtype=tf.float32)
-    numerator = tf.reduce_sum(y_true * y_pred, axis=[1, 2, 3]) * 2 + EPS
-    denominator = (
-        tf.reduce_sum(y_true, axis=[1, 2, 3])
-        + tf.reduce_sum(y_pred, axis=[1, 2, 3])
-        + EPS
+    numerator = tf.reduce_sum(y_true * y_pred, axis=[1, 2, 3]) * 2
+    denominator = tf.reduce_sum(y_true, axis=[1, 2, 3]) + tf.reduce_sum(
+        y_pred, axis=[1, 2, 3]
     )
-    return numerator / denominator
+    return tf.math.divide_no_nan(numerator, denominator)
```

Comment:
isn't this dangerous for hiding potential bugs?

Reply:
no, it's actually safer as 0/0 is not defined, so it can be 0 or anything.